.github/jobs/configure-checks/all.bats

```diff
@@ -203,258 +433,334 @@ compile_assertions_finished () {
-  assert_line "Warning: base URL is unconfigured; generating team documentation will"
-  assert_line "not work out of the box!"
-  assert_line "Rerun configure with option '--with-baseurl=BASEURL' to correct this."
-  assert_line " * website base URL....: https://example.com/domjudge/"
-  assert_line " * documentation.......: /opt/domjudge/doc"
-  run run_configure "--with-baseurl=https://contest.example.org"
-  assert_line "checking baseurl... https://contest.example.org"
-  refute_line "Warning: base URL is unconfigured; generating team documentation will"
-  refute_line "not work out of the box!"
-  refute_line "Rerun configure with option '--with-baseurl=BASEURL' to correct this."
-  assert_line " * website base URL....: https://contest.example.org"
+@test "Build default [root/root] (effective host does both domserver & judgehost)" {
+    build_default root root
 }
 
-@test "Change users" {
-  setup
-  run run_configure
-  assert_line " * default user........: domjudge-bats-user"
-  assert_line " * runguard user.......: domjudge-run"
-  assert_line " * runguard group......: domjudge-run"
-  assert_regex "^ \* webserver group\.\.\.\.\.: (www-data|apache|nginx)$"
-  run run_configure "--with-domjudge-user=other_user --with-webserver-group=other_group --with-runuser=other-domjudge-run --with-rungroup=other-run-group"
-  refute_line " * default user........: domjudge-bats-user"
-  refute_line " * runguard user.......: domjudge-run"
-  refute_line " * runguard group......: domjudge-run"
-  for group in www-data apache nginx; do
-    refute_line " * webserver group.....: $group"
-  done
-  assert_line " * default user........: other_user"
-  assert_line " * runguard user.......: other-domjudge-run"
-  assert_line " * runguard group......: other-run-group"
-  assert_line " * webserver group.....: other_group"
+# Cleanup for next runs
+rm -rf /opt/domjudge >/dev/zero
+make clean >/dev/zero
+
+@test "Build default [user/root] (effective host does both domserver & judgehost)" {
+    build_default "$u" root
 }
 
-@test "No docs" {
-  setup
-  run run_configure
-  assert_line " * documentation.......: /opt/domjudge/doc"
-  run run_configure --enable-doc-build
-  assert_line " * documentation.......: /opt/domjudge/doc"
-  run run_configure --disable-doc-build
-  assert_line " * documentation.......: /opt/domjudge/doc (disabled)"
-}
-
-@test "Build default (effective host does both domserver & judgehost)" {
-  if [ "$distro_id" = "ID=fedora" ]; then
-      # Fails as libraries are not found
-      skip
-  fi
-  setup
-  run run_configure
-  assert_line " * domserver...........: /opt/domjudge/domserver"
-  assert_regex "^ \* webserver group\.\.\.\.\.: (www-data|apache|nginx)$"
-  assert_line " * judgehost...........: /opt/domjudge/judgehost"
-  assert_line " * runguard group......: domjudge-run"
-  run make domserver
-  assert_success
-  run make judgehost
-  assert_success
+# Cleanup for next runs
+rm -rf /opt/domjudge &>/dev/zero
+make clean &>/dev/zero
```

Comment:
Can this be a test setup/teardown action? Adding it here manually makes this code fragile.